### PR TITLE
smoke test case about service "out of order"  of roommanager

### PR DESCRIPTION
**Pull request by attivanm**
smoke test case in the folder Smoke
outOfOrderAPI.js in the folder lib

update file endPoint in the folder config

### CODE REVIEW

**Comment by attivanm** [comment]
refactored the smoke test  out-of-order 
refactored the out-of-orderAPI
created the roomsAPI  
update endpoint 
update config
update the util

File: config/config.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+	"url" : "https://172.20.208.27:4040",
+	"urlmongodb" : "mongodb://172.20.208.27:27017/roommanager",
 	"timeOut" : 4000,
 	"userAccountJson" : {
-		"username": "test",
-	  	"password": "Client123",
-	  	"authentication": "local"
+		"username": "forest1\\admin.ivan",
+	  	"password": "Control123",
+	  	"authentication": "ldap"
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: This is for your personal use, change it please

File: config/config.json [hunk 2]
```diff
@@ -16,8 +17,14 @@
...
 		"meetingsAPI" : "/lib/meetingsAPI",
 		"util" : "/util/util",
-		"roomManagerAPI" : "/lib/roomManagerAPI"
+		"roomManagerAPI" : "/lib/roomManagerAPI",
+		"roomsAPI" : "/lib/roomsAPI"
 	},
 	"tokenHeader" : "Authorization",
-	"tokenPrefix" : "jwt "
+	"tokenPrefix" : "jwt ",
+	"nameId":{
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: Move this to another class

File: tests/CRUD/outOfOrder.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+describe('CRUD test about out of order', function () {
+    
+	/**
+	 * Pre condition to execute the set Test Cases.
+	 * Obtain a token to an user account setting in the config.json file,
+	 * Get the token of room manager
+	 */
+
+	before('Before Set',function (done) {
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/outOfOrder.js]: Maybe you can use a name more descriptive

File: tests/CRUD/outOfOrder.js [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+					token = res.body.token;
+					done();
+				});
+		});
+		 /**
+		 * @description: Pre condition to execute the set Test Cases.
+		 *return an endpoint by roomId and serviceId of a room out of order with the roomId
+		 * @res: res an endpoint with the roomId
+		 */	 
+	before('Before set',function (done) {
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/outOfOrder.js]: Same

File: tests/CRUD/outOfOrder.js [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+		roomManagerAPI.get(RoomEndPoint,function(err,res){
+			room = res.body[0];
+			endPoint1= util.stringReplace(outOfOrderbyIDEndPoint,config.nameId.serviceId,res.body[0].serviceId)
+			endPointOutOfOrder= util.stringReplace(endPoint1,config.nameId.roomId,res.res.body[0]._id)
+			done();
+		});
+	});
+
+	this.timeout(timeout);
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/outOfOrder.js]: This line should be at the beginning of the describe right?

File: tests/Smoke/outOfOrder.js [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+describe('CRUD test about out of order', function () {
+    
+	/**
+	 * Pre condition to execute the set Test Cases.
+	 * Obtain a token to an user account setting in the config.json file,
+	 * Get the token of room manager
+	 */
+
+	before('Before Set',function (done) {
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/outOfOrder.js]: Same

File: tests/Smoke/outOfOrder.js [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+					});	
+			});
+
+		});
+		describe('Smoke of Delete service about out of order', function () {
+		    /**
+			 * @description: Pre condition to execute the set Test Cases.
+			 * create an out of order in a room spesific by ID
+			 */	  
+			before('defore create the out of order',function (done) {
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/outOfOrder.js]: Misspelling

File: util/util.js [hunk 8]
```diff
@@ -1,6 +1,7 @@
 // util
 var config = require('../config/config.json')
 var resourceConfig = require('../config/resource.json');
+var outOfOrderConfig = require('../config/outOfOrder.json');
```
Comment (AlejandroJeanCarlo) [inline util/util.js]: Use are root directory

**Comment by AlejandroJeanCarlo** [comment]
Please review my observations

**Comment by attivanm** [comment]
Update the Smoke test and CRUD  
update Config.json  and Util.js

**Comment by attivanm** [comment]
CRUD and SMOKE test and update config and util

**Pull request merged**